app/helpers/otp_helper.rb

```diff
@@ -0,0 +1,44 @@
+module OtpHelper
+  def phone_confirmation_instructions
+    if @delivery_method == 'sms'
+      t('instructions.2fa.confirm_code_sms', number: @phone_number)
+    elsif @delivery_method == 'voice'
+      t('instructions.2fa.confirm_code_voice', number: @phone_number)
+    end
+  end
+
+  def alternative_2fa_links
+    case @delivery_method
+    when 'sms'
+      "#{voice_fallback_link}#{totp_option_link}."
```

Comment:
joining strings like this is bad when they contain HTML, IMO. the strings may not be HTML-safe, and then the `==` in the template is "forcing" the result to be HTML safe.

I think we could simplify by making this method return an array of strings that the template iterates over. Then we can make sure HTML is escaped properly in each string.


So like:

```ruby
def alternative_2fa_links
  case @delivery_method
  when 'sms'
    [voice_fallback_link, totp_option_link]
# ...
```

```slim
- alternative_2fa_links.each do |link|
  = link
```